app/models/pipeline_run.rb

```diff
@@ -278,6 +293,52 @@ def db_load_ercc_counts
     update(total_ercc_reads: ercc_counts_array.map { |entry| entry[:count] }.sum)
   end
 
+  def db_load_amr_counts
+    # test files: "s3://idseq-database/test/AMR/amr_processed_results.csv", "s3://idseq-database/test/AMR/amr_summary_results.csv"
+    # amr_results = PipelineRun.download_file_with_retries("s3://idseq-database/test/AMR/amr_processed_results.csv", local_amr_full_results_path, 3)
+    amr_results = PipelineRun.download_file_with_retries(s3_file_for("amr_full_results"), local_amr_full_results_path, 3)
```

Comment:
Problem is that you have an inconsistency in the name of the output. In `s3_file_for`, you define the path for `amr_full_results` (https://github.com/chanzuckerberg/idseq-web/blob/157a91386f645c5139b65ba655b5d60b59ea8866/app/models/pipeline_run.rb#L420), but in output_states it is `amr_counts` (https://github.com/chanzuckerberg/idseq-web/blob/157a91386f645c5139b65ba655b5d60b59ea8866/app/models/pipeline_run.rb#L183). They need to be the same. Your loading errors stem from the fact that the check for the presence of `s3_file_for("amr_counts")` succeeded but downloading `s3_file_for("amr_full_results")` failed. Once you make the names consistent, it won't try to load until the file is ACTUALLY there

Reply:
I see, thanks for pointing this out.